### DISCUSSION TITLE
error hyprland.conf at line 105 (Master-Layout)

### DISCUSSION THREAD

**Discussion by Migueh02** [hyprwm/Hyprland#6760] [Q&A]
Hello everyone, I recently updated between them hyprland and when I performed the reboot, this error appeared on this line.
config error in file /home/xxx/.config/hypr/hyprland.conf at line 105: config.option <master:new_is_master> does not exist

![a](https://github.com/prasanthrangan/hyprdots/assets/149646790/9f22a6bf-1621-417a-b3b5-5fe9d333fd47)

I already disabled it so that the error would not appear on the screen

![image](https://github.com/prasanthrangan/hyprdots/assets/149646790/447971d6-3a04-4204-8a6b-7dbc1d68678a)

**Comment by zakk4223** [comment]
Removed and replacew with 'new_status'. See wiki: https://wiki.hyprland.org/Configuring/Master-Layout/
